benchmark/solver/batch_solver.cpp

```diff
@@ -269,14 +268,21 @@ void solve_system(const std::string &sol_name,
         add_or_set_member(solver_case, solver_name,
                           rapidjson::Value(rapidjson::kObjectType), allocator);
         auto &solver_json = solver_case[solver_name];
-        add_or_set_member(solver_json, "recurrent_residuals",
-                          rapidjson::Value(rapidjson::kArrayType), allocator);
-        add_or_set_member(solver_json, "true_residuals",
-                          rapidjson::Value(rapidjson::kArrayType), allocator);
-        add_or_set_member(solver_json, "implicit_residuals",
-                          rapidjson::Value(rapidjson::kArrayType), allocator);
-        add_or_set_member(solver_json, "iteration_timestamps",
-                          rapidjson::Value(rapidjson::kArrayType), allocator);
+        size_type nbatch = system_matrix->get_num_batch_entries();
+        if (FLAGS_detailed && b->get_size().at(0)[1] == 1 && !FLAGS_overhead) {
```

Comment:
Are we limited to one RHS because it's harder to have arrays of arrays in rapidJSON? I think that's fine for now.

Reply:
Yes, I think one rhs is enough, otherwise we have unnecessary lengthy arrays or wider arrays. It is not too difficult to add if we later decide we want it. It would just need another loop over the number of rhs.